src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue

```diff
@@ -151,6 +176,55 @@ export default {
 		onClick(event) {
 			this.$emit('click', event)
 		},
+
+		handleHover() {
+			this.showActions()
+			this.linkIsHovered = true
+		},
+
+		handleFocus() {
+			this.linkIsFocused = true
+			this.showActions()
+		},
+
+		handleBlur(e) {
+			this.linkIsFocused = false
+		},
+
+		showActions() {
+			if (this.hasActions) {
+				this.displayActions = true
+			}
+		},
+
+		handleMouseleave() {
+			this.linkIsHovered = false
+			if (!this.lockActionsVisibility) {
+				this.displayActions = false
+			}
+
+		},
+
+		handleTab(e) {
+			if (this.linkIsFocused && this.hasActions) {
+				console.debug('preventdefault')
```

Comment:
stray debug ? if we want to keep maybe add a bit more info in the string

Reply:
dropped from this pr